Sources/Sentry/include/SentryViewHierarchy.h

```diff
@@ -6,6 +6,15 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface SentryViewHierarchy : NSObject
 
+/**
+ Get the view hierarchy in a json format.
+ Always runs in the main thread.
+ */
+- (nullable NSData *)appViewHierarchy;
+
+/**
+ Get the view hierarchy in a json format.
+ */
 - (nullable NSData *)fetchViewHierarchy;
```

Comment:
`m`: Those two methods do basically the same, except one runs on the main thread and the other not. I would be great, if their name would reflect that. For example, we could rename them to `fetchViewHierarchyOnMainThread` and `fetchViewHierarchyOnBackgroundThread` to clearly state the difference. The same applies to screenshots.